src/lib/game/world/map.js

```diff
@@ -86,6 +92,10 @@ class Map extends THREE.Group {
           }
 
           this.add(m2);
+
+          if (this.world !== null && m2.billboards.length > 0) {
+            this.world.addBillboardedM2(m2);
+          }
```

Comment:
I'm not positive, but typically movement data from the server is only position (x, y, z), orientation (angle), and map ID. Animations, I believe, are almost entirely controlled by the client, with the exception of playing animations in reaction to an opcode (such as a spell cast firing, emote being applied to an object, etc)